pkg/controller/migration/convert/bpf.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2022 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package convert
+
+import (
+	"fmt"
+	"strings"
+
+	operatorv1 "github.com/tigera/operator/api/v1"
+	crdv1 "github.com/tigera/operator/pkg/apis/crd.projectcalico.org/v1"
+	"github.com/tigera/operator/pkg/common"
+	"github.com/tigera/operator/pkg/render"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func copyK8sServicesEPConfigMap(c *components) error {
+	// Extract end point host, port from configmap in kube-system namespace
+	cmName := render.K8sSvcEndpointConfigMapName
+	cm := &corev1.ConfigMap{}
+	cmNamespacedName := types.NamespacedName{
+		Name:      cmName,
+		Namespace: "kube-system",
+	}
+	if err := c.client.Get(ctx, cmNamespacedName, cm); err != nil {
+		return fmt.Errorf("Failed read to ConfigMap %q: %s", cmName, err)
```

Comment:
this is a super nit, but error strings generally start lowercase - e.g., `"failed to read" instead of "Failed to read" 

Makes them compose a bit better if error handling higher up the stack composes them.